libspng.rb

```diff
@@ -0,0 +1,42 @@
+class Libspng < Formula
```

Comment:
It looks you've occasionally created a new file instead of adding changes to `Formula/libspng.rb`; could you please apply these changes to `Formula/libspng.rb` and delete this `libspng.rb`

Reply:
Oh good greif, sorry!